src/modules/Auth/features/Login/Login.schema.ts

```diff
@@ -13,7 +13,7 @@ export const loginFormSchema = () => {
 
   return yup
     .object({
-      email: yup.string().required(emailRequired).email(incorrectEmail),
+      email: yup.string().required(emailRequired).matches(EMAIL_REGEXP, incorrectEmail),
```

Comment:
let's move `yup.string().required(emailRequired).matches(EMAIL_REGEXP, incorrectEmail)` in emailSchema shared variable here and below?

Reply:
UPD: or maybe make it more flexible 
`getEmailSchema({required = true, message }: {required?: booleam, message: string})`